grails-app/views/inventoryItem/_showPendingInboundStock.gsp

```diff
@@ -63,7 +63,7 @@
                                 </g:link>
                             </g:else>
                         </td>
-                        <td class="center">
+                        <td class="center" style="word-break: break-all;">
```

Comment:
Is it ok to break names in the middle of a word? Perhaps the `break-word` instead of `break-all` would be better?

Reply:
You're right. I've tried break-word but in a bad place and assumed it's not working. I will change it.